test/integration_test.js

```diff
@@ -167,10 +167,8 @@ describe("Integration tests", () => {
           expect(cleanRecord(syncResult.published[0])).eql(testData.localUnsynced[0]);
         });
 
-        it("should mark local records as synced", () => {
```

Comment:
Yes, in the [very next test](https://github.com/Kinto/kinto.js/blob/sync-result-updated-list/test/integration_test.js#L174-L187).